src/devices/src/virtio/block/request.rs

```diff
@@ -380,7 +380,7 @@ impl Request {
                 ProcessingResult::Executed(res.user_data.finish(mem, Ok(res.count)))
             }
             Err(e) => {
-                if e.error.is_full_sq() {
+                if e.error.is_throttling_err() {
```

Comment:
The call to `submit_and_wait_all()` should never return `EBUSY` since the entire ring can't contain a number of CQEs that's greater than the capacity of the CQueue. We check this when pushing ops.

Reply:
You're right. The check you added solves the scenario I was thinking about 👍🏻